package.json

```diff
@@ -6,11 +6,13 @@
   "dependencies": {
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
+    "@material-ui/system": "^4.12.2",
```

Comment:
why is it useful to have this package? I see that it is kind of used in `pages/nutrition/index.jsx` but seems to be importing a non used `flexbox` (what would be the advantage of using  that flexbox compared to css defined flexbox?)